cake-recipe.txt

```diff
@@ -7,7 +7,7 @@ Ingredients:
  - 1 teaspoon salt
  - 2 eggs
  - 1 cup milk
- - 1/2 cup vegetable oil
+ - 1/2 cup canolsa oil
```

Comment:
As I dont have Sarah to comment on this - I will comment myself. U misspelled "canola".